test/core/parser/segments/segments_file_test.py

```diff
@@ -6,14 +6,14 @@
 
 
 @pytest.fixture(scope="module")
-def raw_seg_list(generate_test_segments):
+def raw_segments(generate_test_segments):
     """Construct a list of raw segments as a fixture."""
     return generate_test_segments(["foobar", ".barfoo"])
```

Comment:
Isn't a fixture like this already defined in the conftest.py for segments?

Reply:
Ohh, good point. Will remove it.